### ISSUE TITLE
Vertical bias/glitches in isochrones

### ISSUE THREAD

**Issue by abyrd** [conveyal/browsochrones#25]
This could accurately reflect the output data, but it looks so systematic across the whole region that I suspect it's a glitch in the marching squares algorithm. There are elongated vertical notches in the vector isochrones following along the vertical edges of the grid cells. 

![screen shot 2016-02-03 at 17 05 35](https://cloud.githubusercontent.com/assets/112871/12788139/b8fbd5e4-ca98-11e5-9518-637cd0c95859.png)

**Comment by abyrd** [comment]
Here's another image that might give a clue about the nature of the problem. We've got a nice smooth edge here on the isochrone because this area is over water:

![image](https://cloud.githubusercontent.com/assets/112871/14348369/4e7b417c-fcbc-11e5-9cae-359096221f47.png)

Judging by this and the other image above, we've got notches exactly one pixel high occurring right on the pixel boundaries.

It's also worth noting that all those pixels out over the water should be considered inaccessible, but that's dealt with elsewhere in an R5 ticket.
